pymongo/bulk.py

```diff
@@ -377,7 +407,7 @@ def execute_op_msg_no_results(self, sock_info, generator):
                 run.idx_offset += len(to_send)
             self.current_run = run = next(generator, None)
 
-    def execute_command_no_results(self, sock_info, generator):
+    def execute_command_no_results(self, sock_info, generator, write_concern=None):
```

Comment:
Let's make write_concern a required argument since we always pass it.

Reply:
Done